constraint_samplers/test/pr2_arm_kinematics_plugin.cpp

```diff
@@ -105,11 +105,13 @@ int PR2ArmIKSolver::CartToJnt(const KDL::JntArray& q_init,
                               const KDL::Frame& p_in,
                               KDL::JntArray &q_out)
 {
+  bool verbose = false;
```

Comment:
please make that `const bool verbose` everywhere.
This makes it clear it's no runtime option.